sphinx/config.py

```diff
@@ -71,6 +71,7 @@ class Config(object):
         templates_path = ([], 'html'),
         template_bridge = (None, 'html', [str]),
         keep_warnings = (False, 'env'),
+        suppress_warnings = ([], 'env', [list]),
```

Comment:
Same caveat as for other PRs:  Do not add `[list]` here, it is unnecessary from the default value.